src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -90,6 +90,7 @@ protected function describeRoute(Route $route, array $options = array())
             array('Class', \get_class($route)),
             array('Defaults', $this->formatRouterConfig($route->getDefaults())),
             array('Options', $this->formatRouterConfig($route->getOptions())),
+            array('Condition', ('' !== $route->getCondition() ? $route->getCondition() : 'NO CONDITION')),
```

Comment:
i'd prefer the empty string here, like `Host Regex`

Reply:
ok 👍